.github/workflows/docs-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+name: "Release Docs: package the docs and publish to Docs the repo"
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+concurrency:
+  # Note that people may spam the post-commit pipeline on their branch, and
+  # we have this docs pipeline in the post-commit pipeline, then people
+  # would have to wait until the previous one fully completes. That may be
+  # ok because each post-commit pipeline definitely takes more than 30 min
+  group: "pages-${{ github.ref }}"
+  cancel-in-progress: false
+
+jobs:
+  package-push-docs:
+    strategy:
+      # Do not fail-fast because we need to ensure all tests go to completion
+      # so we try not to get hanging machines
+      fail-fast: false
```

Comment:
Likely don't need `fail-fast` here for the reason listed, but I'm wondering if this has anything to do with `concurrency`. I'm also gonna guess so